src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -67,7 +67,7 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ logType }) => {
   } = useLogDownloader(downloadURL, logType);
 
   useEffect(() => {
-    if (data) {
+    if (data && !isLoading) {
```

Comment:
The fetch becomes aborted sometimes because once the data is loaded from the initial `useLogDownloader` query the `useEffect` runs and calls `ingestLines`. `ingestLines` will then close the loading page and redirect to the log display page. If the fetch in the `useFetch` hook is still running it will be aborted because the component is no longer mounted.  The [`abortController`](https://github.com/evergreen-ci/parsley/blob/8b1b28e3389bc88cd9bbd006aa7069f83bd73b84/src/hooks/useFetch/index.ts#L29) manages that. 
By adding a check to ensure we are not actively loading something in the background we can prevent ingesting lines until we have fetched all of the data.

Reply:
To demonstrate here's an example where I took the logkeeper metadata request and slowed it down and you can see how the app waits for it to complete before changing pages. You can see a lot of data is missing because it didn't wait for the request in time.
Before:

https://github.com/evergreen-ci/parsley/assets/4605522/1cb51abb-55e5-4cba-9560-f756242c775e


After:
https://github.com/evergreen-ci/parsley/assets/4605522/503e3529-aaef-4996-99e1-99fdab526894